stair-carousel/index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Dropdown Animation</title>
+
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div id="carousel">
+      <!-- Use of classnames
+      hideleft = img -3 is totally hidden
+      prevSecond = img -2 is reduced and set back
+      prev = img -1 is reduced and set back
+      selected = img is centered and at the first plan
+      next = img +1 is enhanced and brought to the fore
+      nextSecond = img +2 is unhidden and brought to the fore
+      hideRight = img +3 is totally hidden
+    -->
+      <a class="selected" onclick="FindPos(this)" href="javascript:void(0);"
```

Comment:
Any reasons to use an anchor tag `<a />` here if we are making them a dead link in the end?